tests/unit/config/test_converter_telemetry_wandb.py

```
@@ -0,0 +1,176 @@
+        assert cfg.run_name == "run"
+
+    @pytest.mark.parametrize(
+        "field,value",
+        [
+            param("entity", "team", id="entity"),
+            param("run_name", "run", id="run_name"),
+            param("tags", ["a"], id="tags"),
+        ],
+    )  # fmt: skip
+    def test_secondary_without_project_raises(self, field: str, value: object) -> None:
+        with pytest.raises(ValidationError, match="wandb.project"):
```

Ruff (0.15.15)

[warning] Pattern passed to `match=` contains metacharacters but is neither escaped nor raw

(RUF043)